python/pdo/common/keys.py

```diff
@@ -30,25 +31,21 @@ def generate_txn_keys(ledger_type=os.environ.get('PDO_LEDGER_TYPE')):
     """ txn_keys are used to sign register_enclave transaction.
     The format is based on the ledger type"""
 
-    if ledger_type == 'sawtooth':
-        return TransactionKeys()
-    elif ledger_type == 'ccf':
+    if ledger_type == 'ccf':
         return ServiceKeys.create_service_keys()
     else:
```

Comment:
related to my comment above

Reply:
first... CCF will not be our only ledger. be careful about removing too much. second... i believe that the exchange, digital asset and classification contracts use the existing transaction keys. we can have the discussion as to whether this class is necessary, but for now it should stay in as is.